README.md

```diff
@@ -20,5 +20,81 @@ Things you may want to cover:
 * Services (job queues, cache servers, search engines, etc.)
 
 * Deployment instructions
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
+# Budget-app project
+
+> This is a ruby on rails mobile web application where you can manage your budget: you have a list of payments associated with a category, so that you can see how much money you spent and on what.
+
+# Video link
+Here is the [video link](https://www.loom.com/share/50871a7a78944e64bc78acc36f6356bc) for the project.
+
```

Comment:
- [ ] Kindly make sure to deploy the app and add a link in the readme.

Reply:
https://www.loom.com/share/a126415c78d2459d8b70c738e04fe14a

Reply:
@OkoyeCharles You can deploy your development branch with Heroku. The document microverse provided for deploying will be helpful.

Reply:
Okay I will look into it

Reply:
@sja-thedude Thanks very much for highlighting the above changes as you have helped me move my project one step to perfection.
Once again than you